### PR TITLE
[EPMRPP-49053] Getting parameters from examples

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: src/main/java/com/epam/reportportal/cucumber/Utils.java [hunk 1]
```diff
@@ -299,6 +305,25 @@ public static TestCaseIdEntry getTestCaseId(TestStep testStep, String codeRef) {
...
 	}
 
+	static List<ParameterResource> getParameters(List<cucumber.api.Argument> arguments, String text) {
+		List<ParameterResource> parameters = Lists.newArrayList();
+		Optional<String> parameterName = Optional.empty();
+		Matcher matcher = Pattern.compile(PARAMETER_REGEX).matcher(text);
+		if (matcher.find()) {
+			parameterName = Optional.of(text.substring(matcher.start() + 1, matcher.end() - 1));
+		}
+		if (!arguments.isEmpty() && parameterName.isPresent()) {
```
Comment (HardNorth) [inline src/main/java/com/epam/reportportal/cucumber/Utils.java]: The same as for each cucumber agent your code does not take into account many parameters.

File: src/test/java/com/epam/reportportal/cucumber/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class UtilsTest {
```
Comment (HardNorth) [inline src/test/java/com/epam/reportportal/cucumber/UtilsTest.java:18]: Please also add an integration test to ensure you bypass parameter to the client.

**Review by HardNorth — CHANGES_REQUESTED** [review]

File: src/main/java/com/epam/reportportal/cucumber/Utils.java (hunk 1 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/reportportal/cucumber/Utils.java]: fixed

File: src/test/java/com/epam/reportportal/cucumber/UtilsTest.java (hunk 2 above)
Comment (ihar-kahadouski) [reply src/test/java/com/epam/reportportal/cucumber/UtilsTest.java:18]: added

**Review by HardNorth — APPROVED** [review]

**Pull request merged**